app/code/Magento/Store/Ui/Component/Listing/Column/AbstractNameColumn.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Store\Ui\Component\Listing\Column;
+
+/**
+ * Class AbstractNameColumn
+ */
+abstract class AbstractNameColumn extends \Magento\Ui\Component\Listing\Columns\Column
```

Comment:
I believe you reused the code from other Magento module. But not all the legacy code currently fits Magento Technical Guideline (https://devdocs.magento.com/guides/v2.2/coding-standards/technical-guidelines.html#2-class-design). Please try to reimplement this using Composition if possible